src/accessors/IExecutionResult.ts

```diff
@@ -0,0 +1,11 @@
+import { ILogMessage } from './ILogMessage';
+/**
+ * All executions of pre- and post-exits are enabled to perform the modifications
+ * they aim at themselves. This interface provides a mechanism to propagate information
+ * about whether this execution was actually successful
+ */
+
+export interface IExecutionResult {
+    failed: boolean;
+    logMessages: Array<ILogMessage>;
```

Comment:
The messages shall not influence the framework behavior - only the `failed` should be interpreted. Still, if the method failed, a `logMessage` would be expected. 
```javascript
if(failed && !logMessages.filter(message=>message.severity === LogMessageSeverity.ERROR)){
console.error('Rocketlet contract violation: A failed execution has to provide at least one error message');
}
```